tests/src/Functional/PermissionTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
```

Comment:
Is it possible to have a copyright / license header like in the other files? (feel free to set the copyright value to what you feel is appropriate)